DEVELOPER.md

```diff
@@ -59,10 +59,13 @@ Then install all packages necessary to run the tests either by executing [instal
 
 Intel fortran can be used to compile MODFLOW 6 and associated utilities and generate distributable files (if not using gfortran).
 Download the Intel oneAPI HPC Toolkit: https://software.intel.com/content/www/us/en/develop/tools/oneapi/hpc-toolkit/download.html
+Documentation describing how to set the intel envrionment variables can be found [here](https://www.intel.com/content/www/us/en/develop/documentation/oneapi-programming-guide/top/oneapi-development-environment-setup.html).
 
 ### LaTeX (optional)
 
 [LaTeX](https://www.latex-project.org/) which is used to generate the MODFLOW 6 release notes and Input/Output documents (docs/mf6io/mf6io.nightlybuild).
+JLM: should this be more specific? I even believe the installs are in .github/workflow
+JLM: what about doxygen+graphivz? looks like only linux installs
```

Comment:
This should be removed

Reply:
I did doxygen and graphviz via conda.

Reply:
I think I may have used macports for these